Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -132,4 +182,14 @@ def test_alerts_in_sentinel(self):
 
 if __name__ == '__main__':
     os.chdir(os.path.dirname(os.path.abspath(__file__)))
-    unittest.main()
+    suite = unittest.TestSuite()
+
+    # Add all tests except the ones we want to skip
+    for test in unittest.defaultTestLoader.getTestCaseNames(TestCohesity):
+        if test not in (
+            'test_cohesity_close_helios_incident',
```

Comment:
Please add a TODO comment that these have to be removed at the end. A typical test workflow should be

- Create a new workspace
- Install function apps
- Install playbooks
- Wait for the incidents to come in.
- Run all these tests.

Reply:
done